.github/workflows/ci.yaml

```
@@ -590,6 +590,31 @@ jobs:
           projectToken: 695c25b6cb65
           workingDir: "./site"
 
+  offlinedocs:
+    runs-on: ${{ github.repository_owner == 'coder' && 'buildjet-8vcpu-ubuntu-2204' || 'ubuntu-latest' }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          # Required by Chromatic for build-over-build history, otherwise we
+          # only get 1 commit on shallow checkout.
+          fetch-depth: 0
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.20.1
+      - run: yarn install
+        working-directory: ./offlinedocs
+      - run: yarn export
+        working-directory: ./offlinedocs
+      - name: Tar files
+        run: tar -cvf docs.tar offlinedocs/out
+      - name: Upload Artifact
```

```suggestion
      - name: Upload Artifact
      	if: github.ref == 'refs/heads/main'
```
If you add this then we only upload artifacts when PR is merged with the `main` branch.

---

Also, we may remove it entirely from the ci.yaml and include it in release.yaml. 
It's better if we only build on new releases.

---

Yes, 100%. The idea to make it on the PR, for now, was to test it before adding it to the release file.